src/linagora.esn.calendar/app/app.config.js

```diff
@@ -0,0 +1,20 @@
+require('./core/application-menu-calendar.directive.js');
+
+angular.module('esn.calendar')
+  .config(injectApplicationMenuDirective)
+  .config(registerI18N);
+
+function injectApplicationMenuDirective(dynamicDirectiveServiceProvider) {
+  var dd = new dynamicDirectiveServiceProvider.DynamicDirective(true, 'application-menu-calendar', {priority: 40});
+
+  dynamicDirectiveServiceProvider.addInjection('esn-application-menu', dd);
+}
+
+// TODO: Write tests for this (https://github.com/OpenPaaS-Suite/esn-frontend-calendar/issues/24)
```

Comment:
We never write test for config blocks before ;) AFAIK

Reply:
Yes, you're right.